packages/p2p/src/stores/my-profile-store.js

```diff
@@ -42,6 +46,8 @@ export default class MyProfileStore extends BaseStore {
     @observable should_show_add_payment_method_form = false;
     @observable should_show_edit_payment_method_form = false;
 
+    MODAL_TRANSITION_DURATION = 250;
+
```

Comment:
`MODAL_TRANSITION_DURATION` refers to the animation transition between two different modals as they switch in and out in milliseconds